wallet/src/wallet/changeset.rs

```diff
@@ -220,6 +245,31 @@ impl ChangeSet {
             changeset.network = network.map(Impl::into_inner);
         }
 
+        // Select locked outpoints.
+        let mut stmt = db_tx.prepare(&format!(
+            "SELECT txid, vout, is_locked, expiration_height FROM {}",
+            Self::WALLET_OUTPOINT_LOCK_TABLE_NAME,
+        ))?;
+        let rows = stmt.query_map([], |row| {
+            Ok((
+                row.get::<_, Impl<Txid>>("txid")?,
+                row.get::<_, u32>("vout")?,
+                row.get::<_, bool>("is_locked")?,
+                row.get::<_, Option<u32>>("expiration_height")?,
+            ))
+        })?;
+        let locked_outpoints = &mut changeset.locked_outpoints;
+        for row in rows {
+            let (Impl(txid), vout, is_locked, expiration_height) = row?;
+            let outpoint = OutPoint::new(txid, vout);
+            locked_outpoints
+                .locked_outpoints
+                .insert(outpoint, is_locked);
```

Comment:
If we remove `is_locked` you'd only have locked outpoints in the db table and this would always be "true".
```suggestion
                .insert(outpoint, true);
```